taxi/tests/test_forms.py

```diff
@@ -0,0 +1,96 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.forms import (
+    DriverUsernameSearchForm,
+    CarModelSearchForm,
+    ManufacturerNameSearchForm
+)
+from taxi.models import Manufacturer, Car
+
+DRIVER_LIST_URL = reverse("taxi:driver-list")
+CAR_LIST_URL = reverse("taxi:car-list")
+MANUFACTURER_LIST_URL = reverse("taxi:manufacturer-list")
+
+
+class DriverSearchFormTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="driver",
+            password="driver123",
+            license_number="ATM55555",
+        )
+        self.client.force_login(self.user)
+
+    def test_driver_username_search_form(self):
+        form_data = {"username": self.user.username}
+
+        form = DriverUsernameSearchForm(data=form_data)
+        response = self.client.get(DRIVER_LIST_URL, data=form_data)
+
+        self.assertTrue(form.is_valid())
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, self.user.username)
+
+
+class CarSearchFormTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="driver",
+            password="driver123",
+            license_number="ATM55555",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="BMW",
+            country="Test"
+        )
+        self.car = Car.objects.create(
+            model="Sedan",
+            manufacturer=self.manufacturer
+        )

```

Comment:
You must create two different cars because it does not cover searching. Maybe your code will return all instances?